src/workspaces/features/EditWorkspaceDialog/EditWorkspaceDialog.tsx

```diff
@@ -0,0 +1,89 @@
+import Button from "core/components/Button";
+import Dialog from "core/components/Dialog";
+import Spinner from "core/components/Spinner";
+import useForm from "core/hooks/useForm";
+import { useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { useRouter } from "next/router";
+import { Workspace } from "graphql-types";
+import Textarea from "core/components/forms/Textarea";
+import { useUpdateWorkspaceMutation } from "workspaces/graphql/mutations.generated";
+
+type EditWorkspaceDialogProps = {
+  onClose(): void;
+  open: boolean;
+  workspace: Omit<
+    Workspace,
+    "createdAt" | "updatedAt" | "createdBy" | "memberships" | "countries"
+  >;
+};
+
+type Form = {
+  description: string;
+};
+
+const EditWorkspaceDialog = (props: EditWorkspaceDialogProps) => {
```

Comment:
Since you can only edit the description, I'd prefer to rename this as `WorkspaceDescriptionDialog`

Reply:
Okay